### ISSUE TITLE
Iconics (ImageView) > 3.1.0 crashes on Android 4.1

### ISSUE THREAD

**Issue by kuba2k2** [mikepenz/Android-Iconics#436]
## About this issue

- Briefly describe the issue
When using IconicsImageView in version higher than 3.1.0 (these that support animations), the app works fine on Nougat but crashes on Jelly Bean.
[https://pastebin.com/MRdaWErU](https://pastebin.com/MRdaWErU)
- How can the issue be reproduced / sample code

```xml
<com.mikepenz.iconics.view.IconicsImageView
                android:layout_width="32dp"
                android:layout_height="32dp"
                android:layout_marginStart="24dp"
                android:layout_marginLeft="24dp"
                android:layout_marginTop="48dp"
                app:iiv_color="@color/colorPrimary"
                app:iiv_icon="cmd_school"
                app:iiv_size="32dp" />
```

## Details
- [3.2.1] Used library version
- [androidX 1.1.0] Used support library version
- [28.0.3] Used gradle build tools version
- [AS 3.3.1] Used tooling / Android Studio version
- [MaterialDrawer 6.1.2 - the same Iconics version] Other used libraries, potential conflicting libraries

## Checklist

- [x] Searched for [similar issues](https://github.com/mikepenz/Android-Iconics/issues)
- [ ] Checked out the [sample application](https://github.com/mikepenz/Android-Iconics/tree/develop/app)
- [x] Read the [README](https://github.com/mikepenz/Android-Iconics/blob/develop/README.md)
- [x] Checked out the [CHANGELOG](https://github.com/mikepenz/Android-Iconics/releases)
- [x] Read the [FAQ](https://github.com/mikepenz/Android-Iconics/blob/develop/FAQ.md)
- [x] Checked out the [MIGRATION GUIDE](https://github.com/mikepenz/Android-Iconics/blob/develop/MIGRATION.md)

**Comment by mikepenz** [comment]
Do you use proguard?

**Comment by merlin-zaraza** [comment]
The same for me with debug build which haven't proguard

**Comment by kuba2k2** [comment]
I use proguard but it's disabled on debug builds and anyway, I added some rules for iconics.24 lut 2019 17:10 Mike Penz <notifications@github.com> napisał(a):Do you use proguard?

**Comment by merlin-zaraza** [comment]
@mikepenz Any update on this?

**Comment by zTrap** [comment]
Fixed in #439

**Comment by mikepenz** [comment]
Fixed with v3.2.2

Thanks a lot for the report
